grafana/nginx/nginx.conf

```diff
@@ -0,0 +1,104 @@
+worker_processes 1;
+
+daemon off;
+
+events {
+    worker_connections 1024;
+    use epoll;
+}
+
+http {
+
+    server {
+        listen 0.0.0.0:9090;
+
+        location ~ .* {
+            # auth_basic "Prometheus server authentication";
+            # auth_basic_user_file /etc/nginx/.htpasswd;
+            proxy_pass http://prometheus:9090;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection 'upgrade';
+            proxy_set_header Host $host;
+            proxy_cache_bypass $http_upgrade;
+        }
+    }
+
+    server {
+        listen 0.0.0.0:3000;
+
+        location ~ .* {
+            # auth_basic "Grafana server authentication";
+            # auth_basic_user_file /etc/nginx/.htpasswd;
+            proxy_pass http://grafana:3000;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection 'upgrade';
+            proxy_set_header Host $host;
+            proxy_cache_bypass $http_upgrade;
+        }
+    }
+
+    server {
+        listen 0.0.0.0:9093;
+
+        location ~ .* {
+            # auth_basic "Alertmanager server authentication";
+            # auth_basic_user_file /etc/nginx/.htpasswd;
+            proxy_pass http://alertmanager:9093;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection 'upgrade';
+            proxy_set_header Host $host;
+            proxy_cache_bypass $http_upgrade;
+        }
+    }
+
+    # server {
```

Comment:
Commented code = unused code. Remove or provide further comments why we shouldn't delete this